homework2/Event application.js

```diff
@@ -0,0 +1,16 @@
+function getEventDays(daysFromToday){
```

Comment:
So this file looks like a duplicate, did you forget to remove this?

Reply:
Yes